src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/filtering/FilteredHaplotypeFilter.java

```diff
@@ -56,7 +56,7 @@ public double calculateErrorProbability(final VariantContext vc, final Mutect2Fi
     protected void accumulateDataForLearning(final VariantContext vc, final ErrorProbabilities errorProbabilities, final Mutect2FilteringEngine filteringEngine) {
         // we record the maximum non-sequencing artifact that is not this filter itself
```

Comment:
I think this should get updated to match the code update below. "Non-sequencing, non-germline"?

Reply:
done